src/components/button/Button.tsx

```diff
@@ -29,6 +31,8 @@ export interface ButtonProps {
     setRef?: Function;
     /** size of the button */
     size?: 'large';
+    /** whether to show icon when the button is not responsive or on large screens */
+    showIcon?: boolean;
```

Comment:
This works but is a little awkward because it seems to conflict with `isResponsive`. Mostly because this has no effect if `showIcon` is false and `isResponsive` is true. I wonder if it would be appropriate to have a fallback element prop..but I suppose that was what `responsiveContent` was for which I believe we moved away from